bee-node/src/lib.rs

```diff
@@ -11,6 +11,7 @@ mod core;
 mod entrynode;
 mod fullnode;
 mod local;
+mod pem_file;
```

Comment:
I think I would call this simply `pem` for simplicity

Reply:
`pem` is already the name of the crate we are using to parse these files.

Reply:
maybe `pemf` ? :jk:

Reply:
how about `identity`? This makes the module independent of what we use internally to store that identity.

Reply:
I think I would prefer that